### DISCUSSION TITLE
Congrats - A nice Android TV Launcher

### DISCUSSION THREAD

**Discussion by sorryusernameisalreadytaken** [nielsvanvelzen/tv-launcher#401] [General]
This launcher is absolutely fantastic!

My notes:
- The heart markup for favoriting apps is simple and effective.
- No ads – that's such a big plus!
- The use of the channel feature to preview app content. Some atv launchers did not use that feature.
- No settings: While settings could be nice in the future but I love the minimalist approach of having none at all. Perhaps an idea to leave it like that at all.
- Proper display of non-TV apps with their regular Android icons.
- Access to the ShieldTV settings – super handy on the right upper corner.
- Tuner change: Unfortunately, it’s not working on my Shield TV at the moment (or maybe never?) but makes sense on a e.g. Sony TV

A massive thank you for your effort @nielsvanvelzen ! Open Source projects like this are a gift to the community, and it’s clear that a lot of thought and work went into this launcher. I really hope to see an F-Droid release in the future – especially since F-Droid has no dedicated ATV apps yet, and this project would be a perfect fit.

Thank you again, and keep up the amazing work! 👏

**Comment by nielsvanvelzen** [comment]
Thank you for your kind words. This app started out as an experiment but I'm well aware that there are people using it daily as their default launcher without there being a release yet. There's still some things I want to finish before making the first official release, but hopefully I'll have a release on F-Droid and Google Play in a few months.
